### ISSUE TITLE
Does not catch 'expected 1 selector or at-rule'

### ISSUE THREAD

**Issue by jeff1evesque** [brigade/scss-lint#915]
I have a `vagrant up` [build](https://github.com/jeff1evesque/machine-learning/blob/f7f8fe6a53fe1f7a8ab77cc535a53710ce13851f/Vagrantfile), which is automated via puppet. During a corresponding build, I got an error:

```bash
$ vagrant up
[...UNRELATED BUILD OMITTED...]
==> main: Notice: /Stage[main]/Compiler::Start_browserify/Service[browserify]/ensure: ensure changed 'stopped' to 'running'
==> main: Notice: /Stage[main]/Compiler::Start_imagemin/Service[imagemin]/ensure: ensure changed 'stopped' to 'running'
==> main: Notice: /Stage[main]/Compiler::Initial_compile/Exec[touch-jsx-files]/returns: executed successfully
==> main: Notice: /Stage[main]/Compiler::Initial_compile/Exec[node-sass-initial-compile]/returns: {
==> main: Notice: /Stage[main]/Compiler::Initial_compile/Exec[node-sass-initial-compile]/returns:   "status": 1,
==> main: Notice: /Stage[main]/Compiler::Initial_compile/Exec[node-sass-initial-compile]/returns:   "file": "/vagrant/src/scss/style.scss",
==> main: Notice: /Stage[main]/Compiler::Initial_compile/Exec[node-sass-initial-compile]/returns:   "line": 20,
==> main: Notice: /Stage[main]/Compiler::Initial_compile/Exec[node-sass-initial-compile]/returns:   "column": 2,
==> main: Notice: /Stage[main]/Compiler::Initial_compile/Exec[node-sass-initial-compile]/returns:   "message": "Invalid CSS after \"}\": expected 1 selector or at-rule, was \"a: {\"",
==> main: Notice: /Stage[main]/Compiler::Initial_compile/Exec[node-sass-initial-compile]/returns:   "formatted": "Error: Invalid CSS after \"}\": expected 1 selector or at-rule, was \"a: {\"\n        on line 20 of ../../vagrant/src/scss/style.scss\n>> }\n   -^\n"
==> main: Notice: /Stage[main]/Compiler::Initial_compile/Exec[node-sass-initial-compile]/returns: }
==> main: Error: node-sass /vagrant/src/scss/style.scss /vagrant/interface/static/css/style.css returned 1 instead of one of [0]
==> main: Error: /Stage[main]/Compiler::Initial_compile/Exec[node-sass-initial-compile]/returns: change from notrun to 0 failed: node-sass /vagrant/src/scss/style.scss /vagrant/interface/static/css/style.css returned 1 instead of one of [0]
==> main: Notice: /Stage[main]/Webserver::Service/File[/etc/init/start_gunicorn.conf]: Dependency Exec[node-sass-initial-compile] has failures: true
==> main: Warning: /Stage[main]/Webserver::Service/File[/etc/init/start_gunicorn.conf]: Skipping because of failed dependencies
==> main: Notice: /Stage[main]/Webserver::Start/Service[start_gunicorn]: Dependency Exec[node-sass-initial-compile] has failures: true
==> main: Warning: /Stage[main]/Webserver::Start/Service[start_gunicorn]: Skipping because of failed dependencies
==> main: Notice: Applied catalog in 824.24 seconds
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

However, the corresponding [`scss-lint`](https://travis-ci.org/jeff1evesque/machine-learning/builds/282074682#L1113-L1116), executed via travis ci did not indicate any kind of warning, or error. Is my `scss-lint` gem package, outdated at [`0.38.0`](https://github.com/jeff1evesque/machine-learning/blob/e533746a4847ff4dc1317d280de9410555978d90/.travis.yml#L20), or is this a check that should already exist?

**Comment by jeff1evesque** [comment]
Never mind, had `VendorPrefix` flagged off during lint.
